parsers/uniprot/uniprot_test.go

```diff
@@ -0,0 +1,29 @@
+package uniprot
+
+import (
+	"fmt"
+	"testing"
+)
+
+func ExampleReadUniprot() {
+	entries, _, _ := ReadUniprot("data/uniprot_sprot_mini.xml.gz")
+
+	var entry Entry
+	for singleEntry := range entries {
+		entry = singleEntry
+	}
+	fmt.Println(entry.Accession[0])
+	// Output: O55723
+}
+
```

Comment:
Can you add an example test form ParseUniprot?

Reply:
Done!